setup.py

```diff
@@ -437,19 +442,22 @@ def get_extensions():
         ]
 
         generator_flag = []
+        arch_flag = []
+        for arch in os.getenv("PYTORCH_ROCM_ARCH", "native").split(';'):
```

Comment:
One downside I see here is that when we have multiple value in PYTORCH_ROCM_ARCH we will build cpp extensions for each arch, even though we might not run them on all arches. It will slow build time, grow executable size. We could workaround by checking for `HIP_ARCHITECTURES` instead of `FORCE_ROCM` and `PYTORCH_ROCM_FLAGS` in L423:L427